packages/demo/src/components/hook-cards/use-flow-revertible-random-card.tsx

```diff
@@ -0,0 +1,150 @@
+import {useFlowRevertibleRandom} from "@onflow/react-sdk"
+import {DemoCard} from "../ui/demo-card"
+import {useDarkMode} from "../flow-provider-wrapper"
+import {PlusGridIcon} from "../ui/plus-grid"
+import {ResultsSection} from "../ui/results-section"
+
+const IMPLEMENTATION_CODE = `const { 
+  data: randomNumber, 
+  isLoading, 
+  error, 
+  refetch 
+} = useFlowRevertibleRandom({
+  query: { enabled: true }
+})`
+
+export function UseFlowRevertibleRandomCard() {
+  const {darkMode} = useDarkMode()
+  const {
+    data: randomResults,
+    isLoading,
+    error,
+    refetch,
+  } = useFlowRevertibleRandom({
+    max: "1000000000",
+    count: 1,
+  })
+
+  return (
+    <DemoCard
+      id="flow-revertible-random"
+      title="useFlowRevertibleRandom"
+      description="Generate cryptographically secure random numbers on the Flow blockchain that are deterministic and safe for smart contracts."
```

Comment:
I would be clearer about what types of use cases this is actually good (see the API reference) and then link to the docs on revertible random for more sophisticated use cases.